woocommerce/woocommerce-gateway-stripe
# Adds Get more payment method link at the bottom of payment methods

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1894

# Description
Adds a link at the bottom of the payment methods. Clicking on the link will open up a new tab to Stripe's dashboard. 

# Testing instructions
1. Make sure you have `WC_Stripe_Feature_Flags::is_upe_preview_enabled()` returning `true`.
2. Go to `wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe`
3. Confirm "Get more payment methods" button show up and open a new tab when clicked.
![image](https://user-images.githubusercontent.com/572862/134212594-91462db3-755c-4285-af74-d11ea0da4637.png)
